src/library/itunes/itunesfeature.h

```diff
@@ -31,6 +31,11 @@ class ITunesFeature : public BaseExternalLibraryFeature {
 
     TreeItemModel* sidebarModel() const override;
 
+    // This is called form the ITunesXMLImporter thread
+    bool isImportCanceled() {
+        return m_cancelImport.load();
```

Comment:
Even though this is a trivial implementation, should we move this to the .cpp file?

Reply:
I prefer in the header, its so simple and being able to quickly check whether the method is actually threadsafe has value too.